pom.xml

```diff
@@ -42,6 +42,7 @@
     <docker.tag.long>${docker.tag.prefix}9.${jetty9.minor.version}-${maven.build.timestamp}</docker.tag.long>
 
     <docker.openjdk.image>openjdk:8</docker.openjdk.image>
+    <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties</gcloud-projectId-file>
```

Comment:
Can you please do it as part of this PR? We only have one other PR open which is not severely affected by this change.